test/board_test.exs

```diff
@@ -1,15 +1,20 @@
 defmodule BoardTest do
   use ExUnit.Case
 
-  test "places a mark on an empty board 2" do
+  test "returns an empty board" do
+    board = Board.empty_board
+    assert !Board.board_full?(board)
+  end
+
+  test "places mark on empty board" do
     assert Board.place_mark([1,2,3,4,5,6,7,8,9], 1) == {:ok, ["X",2,3,4,5,6,7,8,9]}
```

Comment:
Why not use `Board.empty_board` for this? Or maybe even use a `@empty_board Board.empty_board`? Elixir will replace all later references to `@empty_board` before even executing the file.